examples/next/faustwp-getting-started/package.json

```diff
@@ -20,5 +20,10 @@
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "sass": "^1.54.9"
+  },
+  "nextBundleAnalysis": {
+    "budget": 358400,
+    "budgetPercentIncreaseRed": 20,
+    "showDetails": true
```

Comment:
Since we only use this for our internal CI/CD, we shouldn't expose it to the customer. It could add confusion. We should add these options programatically in the GitHub action above:

```
npm pkg set nextBundleAnalysis.budget=358400 nextBundleAnalysis.budgetPercentIncreaseRed=20 nextBundleAnalysis.showDetails=true --json
```